src/utils/useNumber.ts

```diff
@@ -6,8 +6,11 @@ const getLocaleToUse = (locale?: string) => (locale === 'gx' ? 'fr' : locale ||
  * formatter
  */
 export const useNumberFormatter = (): any => {
-  const formatter = (locale?: string): any => {
-    const localeToUse = getLocaleToUse(locale);
+  const formatter = (locale?: string, countryCode?: string): any => {
+    let localeToUse = getLocaleToUse(locale);
+    if (countryCode) {
+      localeToUse = `${localeToUse}-${countryCode}`;
```

Comment:
can you use this function instead? https://github.com/terraware/terraware-web/blob/main/src/strings/locales.ts#L39
(i think it uses the locales we support, and also fixes any issues where the language-country pair is incorrect or unsupported)